src/app/stylesheets/custom/testimonial.less

```diff
@@ -1,4 +1,5 @@
 .ribbon-testimonial {
```

Comment:
Those ribbons are used to display testimonials, not reviews.

<img width="1168" alt="screen shot 2017-01-25 at 2 03 52 pm" src="https://cloud.githubusercontent.com/assets/3941996/22276365/2b47516e-e307-11e6-85a0-4399c684fd26.png">